superdesk/text_checkers/ai/imatrics.py

```diff
@@ -208,6 +216,27 @@ def _analyze(self, data, **params):
             ),
         )
 
+    def search_images(self, items: list) -> dict:
+        """fetch image suggestions"""
+        if not self.base_url or not self.user or not self.key:
+            logger.warning("IMatrics is not configured properly, can't fetch images")
+            return {"result": []}
+        data = items
+        try:
+            r_data = self._search_images(data)
+        except:
```

Comment:
would be useful to log the exception

Reply:
We were told to add filtering of images without src in the server instead, therefore we made a small change after the approval.